scripts/delayed.js

```diff
@@ -150,21 +150,19 @@ function loadPrivacy() {
 
```

Comment:
the new PrivacyJS library does not need to be delayed as it won't load OneTrust every single time. Please make sure this "delayed.js" script is not actually delayed